CHANGELOG_PENDING.md

```diff
@@ -22,6 +22,8 @@ Special thanks to external contributors on this release:
 
 ### FEATURES
 
+- [cli] [#7033](https://github.com/tendermint/tendermint/pull/7033) Add a `rollback` command to rollback to the previous tendermint state in the event of non-determinstic app hash or reverting an upgrade.
```

Comment:
does an app have to be able to rewrite state for the same block? We should document this if so.

Reply:
Yeah as in the app should be able to go back to the previous height and re-execute all the transactions to get the correct hash. Where exactly do you think we should document this?

Reply:
app docs and upgrade.md would be good. Cant think of anywhere else